sample-xml/app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,9 @@
 <resources>
     <string name="app_name">Sample Xml</string>
     <string name="error_generic">Unexpected error</string>
+
+    <string name="home_title_bar">Home</string>
+
+    <string name="second_title_bar">Second</string>
+    <string name="second_id">ID: %s</string>
```

Comment:
Could you please help rename this string to `second_id_title` to align with Compose side 🙏🏼